driver-sync/src/test/functional/com/mongodb/client/AbstractExplainTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+import com.mongodb.ExplainVerbosity;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.model.Filters;
+import org.bson.BsonDocument;
+import org.bson.Document;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static com.mongodb.client.Fixture.getDefaultDatabaseName;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
+
+public abstract class AbstractExplainTest {
```

Comment:
I went for a single test spec and am testing the reactive driver via the sync adapter.

Reply:
Nice - like it. Only nit - use the Junit5 API?

Reply:
I changed the imports to `org.junit.Assert` but other than that I'm not sure what other change to make.  These look like the new reactive streams tests to me.  

What did you have in mind?

Reply:
Looks good.